feature/experimental/policy/ate_tests/README.md

```diff
@@ -0,0 +1,26 @@
+# RT-3.1: Policy based VRF selection
+
+## Testcase summary
+
+Ensure that protocol and DSCP based VRF selection is configured correctly.
+
+## Testcase procedure
+
+*   Configure DUT with 1 input interface and egress VLANs via a single DUT
+    port-2 connected to ATE port-2 corresponding to:
+    *   network-instance “10”, default route to egress via VLAN 10
+    *   network-instance “20”, default route to egress via VLAN 20
+*   Configure DUT and validate packet forwarding for:
+    *   Matching of IPv4 protocol - to network-instance 10 for all input IPv4.
```

Comment:
Can you help add one more scenario: Matching of a specific IPv4 source address - to network-instance 10. No other received IPv4 in network-instance 10.` ? This one was missed in the original doc.

cc: @nflath
cc: @

Reply:
work in progress. updated readme.

Reply:
Completed, please see new PR https://github.com/openconfig/featureprofiles/pull/521 due to some merge issue here.